docs/src/modules/components/MarkdownElement.js

```diff
@@ -570,159 +567,157 @@ const Root = styled('div')(
         marginBottom: theme.spacing(1),
       },
     },
-  }),
-  ({ theme }) => ({
-    [`:where(${theme.vars ? '[data-mui-color-scheme="dark"]' : '.mode-dark'}) &`]: {
-      color: 'rgb(255, 255, 255)',
-      '& :not(pre) > code': {
-        // inline code block
-        color: `var(--muidocs-palette-text-primary, ${darkTheme.palette.text.primary})`,
-        borderColor: alpha(darkTheme.palette.primaryDark[600], 0.6),
-        backgroundColor: `var(--muidocs-palette-grey-900, ${darkTheme.palette.grey[900]})`,
-      },
-      '& strong': {
-        color: `var(--muidocs-palette-grey-200, ${darkTheme.palette.grey[200]})`,
```

Comment:
This's why I hand-picked the variables to make the component context independent.